docs/resources/aws_api_gateway_stages.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About the aws_api_gateway_stages Resource
+platform: aws
+---
+
+# aws\_api\_gateway\_stages
+
+Use the `aws_api_gateway_stages` InSpec audit resource to test properties of multiple AWS ApiGateway Stages.
+
+The AWS::ApiGateway::Stage resource deploys an API Gateway RestAPI resource to a stage so that clients can call the API over the internet. The stage acts as an environment.
```

Comment:
```suggestion
The `AWS::ApiGateway::Stage` resource deploys an API Gateway REST API resource to a stage so that clients can call the API over the internet. The stage acts as an environment.
```